api/libproject/devstart.rb

```diff
@@ -3,6 +3,13 @@
 require "optparse"
 require "tempfile"
 
+def ensure_git_hooks()
```

Comment:
Looks like we'll need a shared place for Ruby scripts a level up?

Reply:
Can we punt on this? My intuition is that there won't be much code sharing, but I would hate to have any duplication that is non-trivial. Can we trust ourselves to keep an eye on this?

Reply:
OK!